### PR TITLE
#S24FSo0T Setup dotenv

### PR DESCRIPTION
**Pull request by Ally4**
#### What does this PR do?

- This PR is to setup the dotenv

#### Description of Task to be completed?

- Creation of the .env file

- Creation of the .env.example file

#### Any background context you want to provide?

- Installation of the dotenv package

#### What are the relevant Trello stories?

- https://trello.com/c/S24FSo0T/40-setup-dotenv

### CODE REVIEW

**Review by Ntwali-Josue — APPROVED** [review]

**Review by Mugema123 — APPROVED** [review]

**Review by TuyizeeAnastase — APPROVED** [review]

**Comment by blaise82** [comment]
I haven't seen any changes to the packages.json file
which means that no new package was added. eg: dotenv package

remember to solve the conflicts

File: .babelrc [hunk 1]
```diff
@@ -1,3 +1,10 @@
...
-}
\ No newline at end of file
+    "presets":["@babel/preset-env"],
+    "plugins": [
+        ["module-resolver", 
+            {
+            "extensions": [".js", ".jsx", ".es", ".es6", ".mjs"]
+          }
+        ]
+    ]
```
Comment (ceelogre) [inline .babelrc]: Do you need this after our discussion?

File: package.json [hunk 2]
```diff
@@ -34,6 +33,7 @@
     "body-parser": "^1.19.0",
     "ci": "^1.0.0",
     "cors": "^2.8.5",
+    "coverage": "^0.4.1",
```
Comment (ceelogre) [inline package.json]: `coverage` should not be in dependencies.

**Review by ceelogre — CHANGES_REQUESTED** [review]

File: .babelrc (hunk 1 above)
Comment (Ally4) [reply .babelrc]: no

**Review by ceelogre — APPROVED** [review]

**Pull request merged**